app/src/test/java/com/github/pockethub/android/util/ConvertUtilsTest.java

```diff
@@ -30,32 +34,54 @@
 	private static final String REPO_NAME_FIRST_PART = "first";
 	private static final String REPO_NAME_SECOND_PART = "second";
 	private static final String REPO_NAME = REPO_NAME_FIRST_PART + "/" + REPO_NAME_SECOND_PART;
-	private static final String REPO_OWNER_LOGIN = "repo_owner_login";
 
-	private Repository repo;
+	private GitHubEvent event;
 
 	@Before
 	public void setup() {
-		User user = User.builder()
-				.login(REPO_OWNER_LOGIN)
-				.build();
-
-		repo = Repository.builder().
-				name(REPO_NAME)
-				.owner(user)
-				.build();
+
+		event = GitHubEvent.builder()
+                .repo(new GitHubEvent.RepoIdentifier() {
```

Comment:
This need to be fixed, I'll get to today. `GitHubEvent.RepoIdentifier` should really have a builder.

There will be a new release today.

Reply:
Version `0.5.1` has a builder for this and is released now.

Reply:
Ok great, a builder will help a lot.